addons/l10n_it_edi/models/account_edi_proxy_user.py

```diff
@@ -12,6 +12,8 @@ class AccountEdiProxyClientUser(models.Model):
     _inherit = 'account_edi_proxy_client.user'
 
     proxy_type = fields.Selection(selection_add=[('l10n_it_edi', 'Italian EDI')], ondelete={'l10n_it_edi': 'cascade'})
+    company_id = fields.Many2one('res.company', string='Company', required=True,
```

Comment:
Just a comment: in `master`, we should ask if other EDIs should have the same default.

Reply:
It's in our Italian module so I think that will only apply to l10n_it_proxy
And in our case it is fine I think :)

Reply:
Yes, but it might be worth to move it to `account_edi_proxy_user` itself if other EDIs are ok with it.

Reply:
I really do not know.
But I think that it is a spacial case in Italy that we do not create a proxy user for branches.